XmlFormatter/src/Settings/Provider/DataStructure/SerializableScope.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections.Generic;
+
+namespace XmlFormatter.src.Settings.Provider.DataStructure
+{
+    /// <summary>
+    /// This class is a data container to for the xml saving
+    /// </summary>
+    public class SerializableScope
+    {
+        /// <summary>
+        /// Name of the scope
+        /// </summary>
+        private string name;
+
+        /// <summary>
+        /// Public access to the scope name
+        /// </summary>
+        public string Name
+        {
+            get => name;
+            set => name = value;
+        }
+
+        /// <summary>
+        /// Namespace of the class this container was created from
+        /// </summary>
+        private string classType;
+
+        /// <summary>
+        /// Public access to the namespace of the class this container was created from
+        /// </summary>
+        public string ClassType
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this an auto-implemented property and remove its backing field.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5052706)